app/Console/Commands/PushProductsMetafields.php

```diff
@@ -70,9 +71,32 @@ public function handle(): void
                             'type' => 'date',
                             'ownerId' => $productData['admin_graphql_api_id'],
                         ]
-                    ]);
+                    ];
                 }
 
+                $tracks = DB::table('imported_tracklist')
+                    ->where('release_code', str_replace('-', '', $migrationOriginData['sku']))
+                    ->get();
+
+                $tracklist = [];
+                foreach ($tracks as $track) {
+                    $tracklist[] = [
+                        'title' => $track->track_title,
+                        'track_number' => $track->track_index,
+                        'duration' => $track->duration
+                    ];
+                }
```

Comment:
:large_blue_circle: You could use `map()` and avoid using `foreach`

```suggestion
                $tracklist = DB::table('imported_tracklist')
                    ->where('release_code', str_replace('-', '', $migrationOriginData['sku']))
                    ->get()
                    ->map(fn (\stdClass $item) => [
                            'title' => $item->track_title,
                            'track_number' => $item->track_index,
                            'duration' => $item->duration
                    ])
                    ->all();
```

Reply:
fixed